scripts/bootstrap.js

```diff
@@ -71,6 +77,9 @@ const tasks = {
     defaultValue: true,
     option: '--core',
     command: () => {
+      log.info(prefix, 'prepublish');
+      spawn('lerna run prepublish -- --silent');
```

Comment:
this won't work without global lerna installed

Reply:
and without some package deps like `babel` as well

Reply:
isn't lerna a local dependency?

Reply:
Hmm, yes, didn't know you can run local deps with `spawn`. The problem is that we effectively `yarn install` two times in this setup.

Reply:
Ok, you can't run bootstrap script without root dependencies anyway